admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,376 @@
+[source,bash]
+----
+oc edit -n openshift-node-problem-detector configmap node-problem-detector
+----
+
+. Add the following flag to enable kernel-monitor, and docker-monitor.json:
+
+[source,yaml]
+----
+ docker-monitor.json: |
+    {
+        "plugin": "journald",""
+----
+
+<1> Sends the output to standard output (stdout).
+<2> Comma-separated path to the plugin configuration files.
+
+To display node problem detector output to standard output (stdout) and standard error (stderr) edit the node problem detector daemonset:
+
+[source,bash]
+----
+# oc edit ds node-problem-detector
+----
+
+. Add the following flag to set the output and enable kernel-monitor, and docker-monitor.jason:
+
+[source,yaml]
+----
+spec:
+  containers:
+  - command:
+    - node-problem-detector
+    - --alsologtostderr=true <1>
+      --log_dir="/tmp" <2>
+    - --system-log-monitors=/etc/npd/kernel-monitor.json,/etc/npd/docker-monitor.json <3>
+----
+
+<1> Sends the output to standard output (stdout).
+<2> Path to the error log.
+<2> Comma-separated path to the plugin configuration files.
+
+
+[[admin-guide-node-problem-detector-fbk]]
+=== Using File-Based Kernel Logging 
+
+The node problem detector supports file-based kernel logging. 
+
+You can use other file-based logging tools, such as link:https://github.com/kubernetes/node-problem-detector/tree/master/pkg/systemlogmonitor#system-log-monitor[System Log Monitor].
+Use the following command with a comma-separated list of paths to system log monitor configuration files:
+
+[source,bash]
+----
+# oc node-problem-detector --system-log-monitors <path to config files>
+----
+
+For example:
+
+[source,bash]
+----
+# oc node-problem-detector --system-log-monitors=config/kernel-monitor.json
+---- 
+
+[NOTE]
+====
+The node problem detector does not detect problems reported by log tools, such as journald or rsyslogd.
+====
+
+The node problem detector starts a separate log monitor for each configuration. You can use different log monitors to monitor different system logs.
+
+[[admin-guide-node-problem-detector-uninstall]]
+== Uninstall the Node Problem Detector
+
+To uninstall the node problem detector:
+
+. Add following options in Ansible inventory file:
++
+[source,yaml]
+----
+[OSEv3:vars]
+openshift_node_problem_detector_state=absent
+----
+
+. Run the following Ansible playbook:
++
+[source,bash]
+----
+# ansible-playbook ~/openshift-ansible/playbooks/openshift-node-problem-detector/config.yml
+----
+////
+
+[[admin-guide-node-problem-detector-standalone]]
+== Starting the Node Problem Detector as Standalone
+
+To run node problem detector as a standalone system daemon, execute the following command, pointing to the IP address of your access API server:
+
+[source,bash]
+----
+# oc node-problem-detector --apiserver-override=http://APISERVER_IP:APISERVER_INSECURE_PORT?inClusterConfig=false
+----
+
+[NOTE]
+====
+Standalone mode uses an insecure API server connection.
+====
+
+[[admin-guide-node-problem-detector-gce]]
```

Comment:
I think we ought to remove this entire section. It's Kube-specific and not really relevant to OpenShift.